build/dockerfiles/skaffold.Dockerfile

```diff
@@ -10,6 +10,8 @@
 
 FROM docker.io/node:18.19.1-alpine3.19
 
+LABEL quay.expires-after=1w
```

Comment:
was not aware of this feature - https://docs.redhat.com/ja/documentation/red_hat_quay/3.3/html/use_red_hat_quay/tag-expiration 
why do we need it?

Reply:
we use this docker file to build images for testing purposes, so we don't need them to be in the registry forever.